contracts/exchange_contract/tests/harness.rs

```diff
@@ -233,6 +233,26 @@ async fn exchange_contract() {
     // Inspect the wallet for alt tokens to be 100
     assert_eq!(total_amount, 100);
 
+    // Return reserve amounts and fee
+    let result = exchange_instance
+        .get_swap_with_minimum(10)
+        .call()
+        .await
+        .unwrap();
+    assert_eq!(result.value.amount, 9);
+    assert!(result.value.has_liquidity);
+    assert_eq!(result.value.reserve, 100);
+    assert_eq!(result.value.fee, 333);
+    let result = exchange_instance
+        .get_swap_with_maximum(10)
```

Comment:
maybe add these values as constants if they are the default

Reply:
just for the tests?

Reply:
might as well, that way anyone else who might happen upon them can understand those values

Reply:
also, some of these you already have constants that represent those values, right?

Reply:
yes will change it thanks